input/blog/2021-02-07-cake-v1.0.0-released.md

```diff
@@ -0,0 +1,319 @@
+﻿---
+title: Cake v1.0.0 released
+category: Release Notes
+author: devlead
+---
```

Comment:
Add `releaseName: 1.0.0` to front-matter to have release 1.0 shown and linked on start page

Reply:
Fixed. Then this should be added to fetch contributors console too, so it's not forgotten when scaffolding.